src/components/inputs/checkbox-input/checkbox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import vars from '../../../../materials/custom-properties';
+
+// accessible input :)
+const Input = styled.input`
+  pointer-events: none;
+  height: 100%;
+  left: 0;
+  opacity: 0.0001;
+  position: absolute;
+  top: 0;
+  width: 100%;
+
+  ${props =>
+    !props.hasError &&
+    `
+  &:focus + div > svg [id$='borderAndContent'] > [id$='border'] {
+    stroke: ${vars.borderColorInputFocus};
+  }`}
+`;
+
+class Checkbox extends React.Component {
+  static displayName = 'Checkbox';
+
+  static propTypes = {
+    isIndeterminate: PropTypes.bool,
```

Comment:
The forwarded prop-types are not noted here. Is that on purpose?

Reply:
Well the idea at the beginning was just to make a super simple component that was just an `<input>` that passed indeterminate state. But yeah, I can refactor it

Reply:
Yeah I see that. But I think it's two things to 1. Have components with inconsistent API and 2. Components which forward any native DOM props.

Reply:
yeah that makes sense, [here](https://github.com/commercetools/ui-kit/pull/718/commits/a6d0a9cd081f35e3364b6286ab97a471c280a015)